benchmark/EstimateMaxCount.t.sol

```diff
@@ -62,30 +61,4 @@ contract EstimateMaxCount is Test {
             console2.log("count: %d and gasUsed: %d and chainId: %d", count, lastGasConsumed, chains[i].chainId);
         }
     }
-
-    /// @notice Estimate the maximum number of tranches allowed in LockupTranched.
-    function test_EstimateTranches() public {
```

Comment:
IMO since the value of `MAX_COUNT` affects both LD and LT, we should keep this function and update bash script to update the minimum of max_segment_count and max_tranche_count. We know that as of now, max_segment_count < max_tranche_count but if we change the logic of tranches in the future (if we make it more advanced), it would be better this way. And bash scripts are fast so I dont see any downside doing that.

Reply:
there is no need to keep unused logic if it's not being used

we know for sure that tranches would cost less gas than segments, due to one less storage variables (exponent) thus safe to remove it.

you are talking about hypothetical cases. how can you know that we won't add more variables in the segments ?

Reply:
> there is no need to keep unused logic if it's not being used

its used to create the minimum of two.

> you are talking about hypothetical cases. how can you know that we won't add more variables in the segments 

Computations also play a role in determining gas usage. I agree I dont know if we add more variables to segments or not and thats why I recommend to take the min of two so we wont have to worry about it at all.

Reply:
Marking it as resolved since as of now, MAX_COUNT is driven by Lockup Dynamic segments.